tests/ZendTest/Db/Adapter/Driver/Pgsql/ConnectionTest.php

```diff
@@ -52,6 +52,19 @@ public function testResource()
     }
 
     /**
+     * Test disconnect method to return instance of ConnectionInterface
+     * @TODO Exception was raised due to invalid resource. Add resource to make this test pass.
+     */
+    public function testDisconnect()
+    {
+        if (! extension_loaded('pgsql')) {
+            $this->markTestSkipped('pgsql extension not loaded');
+        }
+        $this->assertInstanceOf('\Zend\Db\Adapter\Driver\ConnectionInterface', $this->connection->disconnect());
```

Comment:
Since we are testing `return $this` is better to use `assertSame($this->connection, $this->connection->disconnect());`